### ISSUE TITLE
Error Parsing SQL

### ISSUE THREAD

**Issue by jpeyper** [zendesk/maxwell#309]
It's been a while since we saw one of these:

```
2016-04-12T00:01:57,811 INFO  com.google.code.or.net.impl.AuthenticatorImpl: login successfully, user: asiscdc, detail: OKPacket[packetMarker=0,affectedRows=0,insertId=0,serverStatus=2,warningCount=0,message=<null>]
2016-04-12T00:01:58,162 ERROR com.zendesk.maxwell.schema.ddl.MysqlParserListener: (length ( <missing INTEGER_LITERAL> ))
2016-04-12T00:01:58,162 ERROR com.zendesk.maxwell.schema.ddl.SchemaChange: Error parsing SQL: 'create table if not exists audit_payer_bank_details (    username varchar(128) not null, action_type enum('create', 'update', 'delete') not null,        event_time TIMESTAMP default CURRENT_TIMESTAMP(),       id int(11) not null,    employer_id    varchar(255),    abn            varchar(255),    bsb            varchar(255),    account_number varchar(255),    account_title  varchar(255),    employer_name  varchar(255)) engine = ARCHIVE'
2016-04-12T00:01:58,166 INFO  com.zendesk.maxwell.schema.SchemaPosition: Storing final position: null
```

**Comment by osheroff** [comment]
parenthesis but no length in ` CURRENT_TIMESTAMP()`, gotcha.  will fix.

**Comment by osheroff** [comment]
fixed in v1.0.1
